contracts/AccountantImplementation.sol

```diff
@@ -57,23 +65,26 @@ contract AccountantImplementation is FundsRecovery {
     event LoanReturnRequested(bytes32 channelId, uint256 timelock);
     event LoanReturnRequestInvalidated(bytes32 channelId);
     event LoanReturned(bytes32 channelId, address beneficiary, uint256 amount);
-    event ChannelBeneficiaryChanged(bytes32 _channelId, address _newBeneficiary);
+    event ChannelBeneficiaryChanged(bytes32 channelId, address newBeneficiary);
     event FundsWithdrawned(uint256 amount, address beneficiary);
+    event AccountantFeeUpdated(uint16 newFee, uint64 validFromBlock);
 
     /*
       ------------------------------------------- SETUP -------------------------------------------
     */
 
     // Because of proxy pattern this function is used insted of constructor.
     // Have to be called right after proxy deployment.
-    function initialize(address _token, address _operator) public {
+    function initialize(address _token, address _operator, uint16 _fee) public {
         require(!isInitialized(), "have to be not initialized");
         require(_operator != address(0), "operator have to be set");
         require(_token != address(0), "token can't be deployd into zero address");
+        require(_fee <= 5000, "fee can't be bigger that 50%");
```

Comment:
What is 5000 ? 5k tokens? why 5k? How come its 50%?

Reply:
1% = 1.00 = 100 sub procent parts
50% = 50.00 = 5000 subprocent parts

Reply:
We're working only with integers and I want to have possibility to charge 0.05% fee.